packages/devkit/package.json

```diff
@@ -35,7 +35,8 @@
     "semver": "7.3.4"
   },
   "peerDependencies": {
-    "nx": ">= 14 <= 16"
+    "nx": ">= 14 <= 16",
+    "typescript": "^3 || ^4"
```

Comment:
Can we make the peer dependency on `typescript` optional?

I think it should be okay if the user doesn't want to have `typescript` installed.

Reply:
I would argue that it shouldn't be, since it's not optional in `@phenomnomnominal/tsquery`. `tsquery` lists it as a required peer dependency, so `@nrwl/devkit` should do the same or install it.

That being said, making it optional makes the warning go away (even though it probably shouldn't) and this isn't my repo. Let me know how you want me to proceed.